tests/5.0/teams_loop/test_target_teams_loop_allocate.c

```diff
@@ -0,0 +1,259 @@
+//===-------------- test_target_teams_loop_allocate.c---------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+// The tests in this file tests 'teams loop' construct coupled with map()
+// clause in combination with different allocator types
+//
+//===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+/**
+  This is a basic test to demonstrate how allocate can be used with
+  target teams loop construct. Allocate clause is used to allocate a local
+  variable of size int.
+*/
+int testTargetTeamsAllocateCl() {
+  int a[N], b[N], c[N], local = 0;
+  int errors = 0;
+  // Data Inititalize
+  for (int i = 0; i < N; i++) {
+    a[i] = 2*i;  // Even
+    b[i] = 2*i + 1;  // Odd
+    c[i] = 0;
+  }
+  // Execute on target
+#pragma omp target teams map(to: a[0:N], b[0:N]) map(from: c[0:N]) \
+        uses_allocators(omp_cgroup_mem_alloc) \
+        allocate(omp_low_lat_mem_alloc: local) private(local)
+  #pragma omp loop private(local)
```

Comment:
The purpose of this test is to test the `teams loop` combined construct, but this sub-test uses separate `teams` construct and `loop` construct, defeating the purpose of this test.

Reply:
Replaced "teams" with  "teams loop" as suggested...